### ISSUE TITLE
Update to next google-cloud-java release once it's out

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#3500]
Our patches to `google-cloud-java` in https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2281 and https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2283 to fix the transient NIO errors have now been merged into master, and will be part of their next release (which will be the release after `0.22.0`).

We should update to the next release as soon as it's out, to remove our existing dependency on a SNAPSHOT build of `google-cloud-java`.

**Comment by droazen** [comment]
Re-opening

**Comment by droazen** [comment]
Blocked by https://github.com/broadinstitute/gatk/issues/3591
